tests/specs/mcd/flopper-file-pass-rough-spec.k

```diff
@@ -101,14 +101,13 @@ module FLOPPER-FILE-PASS-ROUGH-SPEC
        andBool (#rangeUInt(48, Ttl)
        andBool (#rangeUInt(48, Tau)
        andBool ((#sizeByteArray(CD) <=Int 1250000000)
-       andBool (((ABI_what =/=Int #string2Word("ttl") andBool ABI_what =/=Int #string2Word("tau")) orBool #rangeUInt(48, ABI_data))
```

Comment:
Am I correct that removing this condition makes the claim stronger? Is this intended and this condition isn't needed for verification?

Reply:
Yes, but there was also a problem with the Matching Logic interpretation of `#Or`, where both sides have to be defined in all cases, and the rhs is not always defined.